source/sxl_traffic_light_controller.rst

```diff
@@ -494,12 +494,13 @@ Controller switched on |br|  |br| The controller is active and is not in dark mo
    :loc: H
    :spec: >{\raggedright\arraybackslash}p{0.15\linewidth} p{0.08\linewidth} p{0.13\linewidth} p{0.50\linewidth}
 
-   ============  =======  =================  ============================================================================================================================
-   Name          Type     Value              Comment
-   ============  =======  =================  ============================================================================================================================
-   intersection  integer  [0-255]            Comma separated list of intersections which the status relates to, |br| e.g. "1,2". Use "0" for all intersections of the TLC
-   status        boolean  -False |br| -True  False: Traffic Light Controller in dark mode |br| True: Traffic Light Controller not in dark mode
-   ============  =======  =================  ============================================================================================================================
+   ============  =======  =================================================================================  =================================================================================================================================================================================================================================================
+   Name          Type     Value                                                                              Comment
+   ============  =======  =================================================================================  =================================================================================================================================================================================================================================================
+   intersection  integer  [0-255]                                                                            Comma separated list of intersections which the status relates to, |br| e.g. "1,2". Use "0" for all intersections of the TLC
+   status        boolean  -False |br| -True                                                                  False: Traffic Light Controller in dark mode |br| True: Traffic Light Controller not in dark mode
+   source        string   -operator_panel |br| -calendar_clock |br| -control_block |br| -forced |br| -other  operator_panel: Operator panel is the source |br| calendar_clock: Calendar/clock is the source |br| control_block: Control block is the source |br| forced: Forced due to command from e.g. RSMP |br| other: TLC switched status due other reason
```

Comment:
what status is it in after starting up?

Reply:
Good point. I suggest adding `startup` as a source